src/protagonist/API/Features/Image/Ingest/DeliveryChannelProcessor.cs

```diff
@@ -196,6 +196,16 @@ private async Task AddDeliveryChannelsForMediaType(Asset asset)
             await defaultDeliveryChannelRepository.MatchedDeliveryChannels(asset.MediaType!, asset.Space,
                 asset.Customer);
 
+        if (matchedDeliveryChannels.Any(x => x.Channel == AssetDeliveryChannels.None) &&
+            matchedDeliveryChannels.Count != 1)
+        {
+            throw new APIException("An asset can only be automatically assigned a delivery channel of type 'None' when it is the only one available. " +
+                                   $"Please check your default delivery channel configuration for media type '{asset.MediaType}'")

```

Comment:
Defaults use mediaType + space. I'd either take out `for media type '{asset.MediaType}'"` or included space details too.